### PR TITLE
chore: rename cat photo project

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: utils/preformatted-block-names.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "machine-learning-with-python": "Machine Learning with Python",
   "tensorflow": "TensorFlow",
   "basic-javascript-rpg-game": "Basic JavaScript RPG Game",
-  "basic-html-cat-photo-app": "HTML Cat Photo App",
+  "learn-html-by-building-a-cat-photo-app": "HTML Cat Photo App",
```
Comment (ShaunSHamilton) [inline utils/preformatted-block-names.json:22]: Why are we not changing the block names?

**Review by moT01 — COMMENTED** [review]
I noticed the title here is `Learn HTML By Building A Cat Photo App` (all words capitalized). On the relational database stuff I did not capitalize the `by` or `a` - `Learn HTML by Building a Cat Photo App`. Seems like we could be consistent here? Any preference on which one? Or maybe it isn't a big deal.

File: utils/preformatted-block-names.json (hunk 1 above)
Comment (nhcarrigan) [reply utils/preformatted-block-names.json:22]: Because the block name didn't match the search 🤦🏻

**Comment by nhcarrigan** [comment]
I agree consistency is key, but don't have a preference for which one we use.

**Comment by ShaunSHamilton** [comment]
> Any preference on which one? Or maybe it isn't a big deal.

BiG dEaL! Prepositions and articles are lowercased.

**Review by ShaunSHamilton — APPROVED** [review]
Let us get moving on these 🚢

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**